src/kv/RocksDBStore.cc

```diff
@@ -994,10 +1053,35 @@ void RocksDBStore::RocksDBTransactionImpl::rm_range_keys(const string &prefix,
     }
   } else {
     if (db->enable_rmrange) {
-      bat.DeleteRange(
-	db->default_cf,
-	rocksdb::Slice(combine_strings(prefix, start)),
-	rocksdb::Slice(combine_strings(prefix, end)));
+      if (db->max_items_rmrange) {
+        uint64_t cnt = db->max_items_rmrange;
+        auto it = db->get_iterator(prefix);
+        bat.SetSavePoint();
+        it->lower_bound(start);
+        while (it->valid()) {
+          if (it->key() >= end) {
+            break;
+          }
+          if(!cnt) {
+            bat.RollbackToSavePoint();
+	    bat.DeleteRange(
```

Comment:
tab and spaces.

Reply:
@tchaikov, here if we reach `tabstop`(it is 8), `smarttab` will insert an \<tab\> in front, then we need an extra space, right? I just modify the `tabstop` to 20, is it also ok?

Reply:
@Aran85 the reason i commented on this line is that your were using spaces before `bat.RollbackToSavePoint()` while using <tab> and spaces before `bat.DeleteRange(`. i think that's why github's web interface shows that these two lines are not aligned.

i think we are using tab stop = 8 . see https://github.com/ceph/ceph/blob/master/CodingStyle#L77